### PR TITLE
 CCP-102 chore/front(message): make user icons dynamic in lobbies and admin

### PR DESCRIPTION
**Pull request by djodjo89**

### CODE REVIEW

File: front/src/View/Pages/Inscription.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    }
+
+    public checkForm () : boolean {
+        console.log(this.passwordVerification());
+        if (this.passwordVerification() && this.prenom !== '' && this.nom !== '' && this.pseudo !=='' && this.email !== ''){
+            console.log("Formulaire Correct");
+            return true;
+        }
+        else{
+            console.log("Form pas correct");
```
Comment (Yukitsuo) [inline front/src/View/Pages/Inscription.tsx]: Maybe rephrase the messsage : "Information manquante ou incorecte"

File: front/src/View/Admin/User.tsx [hunk 2]
```diff
@@ -13,11 +14,43 @@ interface UserProps {
 }
 
 class User extends React.Component<UserProps, any> {
+    public constructor(props: any) {
+        super(props);
+
+        this.fillIcon = this.fillIcon.bind(this);
+        this.getIcon = this.getIcon.bind(this);
+    }
+
```
Comment (hippolyte94) [inline front/src/View/Admin/User.tsx]: no state for id ?

File: front/src/View/Admin/User.tsx [hunk 3]
```diff
@@ -30,21 +63,22 @@ class User extends React.Component<UserProps, any> {
...
-                            className={'remove-button minus-icon ml-5 mr-0'}
+                            className={'remove-button minus-icon mr-0'}
                             onClick={this.props.delete}
                         />
                     </div>
                 </div>
                 <div className={'row col-12 mt-2 rounded ml-1 ml-md-0 ml-sm-0 pr-0 pl-1 pl-md-0 pl-sm-0'}>
-                    <div className={'col-3 col-lg-2 col-md-2 col-sm-2 mr-4 mr-lg-0 mr-md-0 mr-sm-0 ml-1 ml-lg-0 ml-md-0 ml-sm-0 pr-0 pl-2 pl-lg-4 pl-md-3 pl-sm-3 text-left'}>
+                    <div className={'row col-2 col-lg-1 col-md-1 col-sm-2 mr-4 mr-lg-0 mr-md-0 mr-sm-0 ml-1 ml-lg-0 ml-md-0 ml-sm-0 pr-0 pl-2 pl-lg-4 pl-md-3 pl-sm-3 text-left'}>
                         <img
```
Comment (hippolyte94) [inline front/src/View/Admin/User.tsx]: we love responsivity

File: front/src/View/Lobby/Message.tsx [hunk 4]
```diff
@@ -2,28 +2,64 @@ import React, {ReactNode} from 'react';
...
     pseudo: string,
+    icon: string,
     activeRemoveButton?: boolean,
     delete?: ((event: React.MouseEvent<HTMLImageElement, MouseEvent>) => void) | undefined,
 }
 
 class Message extends React.Component<MessageProps, {}> {
+    public constructor(props: any) {
+        super(props);
+
```
Comment (hippolyte94) [inline front/src/View/Lobby/Message.tsx]: no state for all the attributes of MessageProps ?

File: front/src/View/Lobby/Messages.tsx [hunk 5]
```diff
@@ -36,7 +38,7 @@ class Messages extends React.Component<MessagesProps, any> {
         // @ts-ignore
         if (undefined === this.props.messages['message']) {
             return this.props.messages.map(
-                (message: any) => this.renderMessage(message['id_message'], message['content'], message['send_date'], message['pseudo'])
+                (message: any) => this.renderMessage(message['id_message'], message['id_user'], message['content'], message['send_date'], message['pseudo'], message['icon'])
```
Comment (hippolyte94) [inline front/src/View/Lobby/Messages.tsx]: attribute key is not used , normal ?
